src/content/docs/style-guide/formatting/code-examples.mdx

```diff
@@ -127,7 +144,46 @@ To format inline code (like `maxSampleTimes` or `TransactionError`), surround th
 
 ## Multi-line code blocks [#code-block]
 
-To format one or more lines of code, insert three backticks ( ```` ``` ```` ) above and below your code. This essentially acts like the HTML `<pre>` tag.
+To format one or more lines of code, insert three backticks ( ```` ``` ```` ) above and below your code. This essentially acts like the HTML `<pre>` tag. 
+
+When you sandwich your code between three backticks, you need to consider both the code layout (indentation) and the syntax highlighting.
+
+### Code layout [#code-layout]
+
+As a technial writer you'll be given code snippets from a variety of languages. To make it easier for your readers to understand code snippets, make sure that lines are indented and grouped in a helpful way.
+
+#### Strict indentation [#strict]
+Some programming and markup languages require specific indentation (for example, Python and YAML). Check with your subject matter expert if you are unsure about whether the snippet you've been given needs to have specific indentation in order to work. You can always run the snippet through an online code prettifier if it gets jumbled when you transfer it from another editing tool.
+
+#### Non-strict indentation [#casual]
+For languages that are casual about indendataion, we recommend that you look for similar examples and follow those patterns. In the case of NRQL, which works with a wide range of indentation and grouping, we recommend you do the following to make it easier to read:
+
+* Capitalize keywords (clauses, functions, and operators), such as FROM, SELECT, FACET, WHERE, AND, OR, AS, SINCE, ORDER BY, JOIN and MAX.
+* The first line should start with FROM, althought queries run fine when you start with SELECT.
+* Left-justify keywords except in the following cases:
+    * If the keyword is closely tied to a preceding keyword (for example, AGO is closely tied to SINCE), so it belongs on the same line.
+    * If the keyword doesn't have any modifiers (it stands alone), you can tack it at the end of a line that already has keywords (For example, TIMESERIES, AUTO, and EXTRAPOLATE).
+    ```sql
+    FROM Transaction
+    SELECT count(*)
+    WHERE appName='interestingApplication'
+    SINCE 60 minutes AGO EXTRAPOLATE
+    ```
+* For keywords inside of parentheses, indent them two spaces. For example, the WHERE keyword is indented two spaces like this:
+    ```sql
+    FROM PageView
+    SELECT count(*)
+    FACET CASES
+    (
+      WHERE duration < 1,
+      WHERE duration > 1 and duration < 10,
+      WHERE duration > 10
+    )
+    ```
+
+### Syntax highlighting [#syntax-highlighting]
+
+We recommend that you add a code fence at the top of your snippet that is sandwiched between backticks. The code fence is simply the name of the language following the first three snippets. See the collapsers below for examples.
```

Comment:
`following the first three snippets` confused me, should this be `the first three backticks`?

Reply:
@austin-schaefer I think this was from an earlier build because I'm no longer seeing this. I had made this correction yesterday.